src/lib/ArrayTextField/index.js

```diff
@@ -7,7 +7,6 @@ import InputAdornment from '@mui/material/InputAdornment'
 import TextField from '@mui/material/TextField'
 import AddIcon from '@mui/icons-material/Add'
 import Chip from '@mui/material/Chip'
-
```

Comment:
This blank space is necessary.
clean code rule 

`order import by a group`

first group: third-party libraries
second group: project libraries (components, config, utils, etc)
third group: files on the same level (styles files or a specific component used only here)

so, these groups should be separated by a blank space